src/containers/Login/Login.js

```diff
@@ -2,13 +2,17 @@ import React, { Component } from 'react'
 import './Login.scss'
 import { loginUser } from '../../actions'
 import { connect } from 'react-redux'
+import { NavLink } from 'react-router-dom'
+import { cleanUser } from '../../helpers/cleanUser'
 
 export class Login extends Component {
     constructor() {
         super()
         this.state = {
             email: '',
-            password:''
+            password:'',
+            id: 10,
```

Comment:
We should take out this hardcoded ID.  Originally we used it to verify our state but now we should be able to delete it.